Documentation/common-problems.md

```diff
@@ -0,0 +1,128 @@
+rook-ceph-mon2-rlxcd                 1/1       Running   0          2d        192.168.123.33   k8-host-0403
+rook-ceph-osd-bg2vj                  1/1       Running   0          2d        192.168.91.177   k8-host-0404
+rook-ceph-osd-mwxdm                  1/1       Running   0          2d        192.168.123.31   k8-host-0403
+```
+
+### Solution
+What is happening here is that the MON pods are restarting and one or more of the CEPH daemons are not getting configured with the proper cluster information. This is commonly the result of not specifying a value for `dataDiskHostPath` in your Cluster CRD.
+
+The `dataDiskHostPath` setting specifies a path on the local host for the CEPH daemons to store configuration and data. Setting this to a path like `/var/lib/rook`, reapplying your Cluster CRD and restarting all the CEPH daemons (MON, MGR, OSD, RGW) should solve this problem. After the CEPH daemons have been restarted, it is advisable to restart the rook-api and rook-tool Pods. 
+
+## Reinstall fails to start all Pods
+
+* Entire Rook.io CRDs have been deleted and the Cluster CRD has been reapplied
+* Rook operator is running
+* Only a partial number of the MON daemons are started
+
+### Further Details
+When attempting to reinstall Rook.io, the rook-operator pod gets started successfully and then the cluster CRD is then loaded. The rook-operator only starts up a single MON (possibly on rare occasion a second MON may be started) and just hangs. Looking at the log output of the rook-operator the last operation that was occuring was a `ceph mon_status`.
+
+Attempting to run the same command inside the MON container shows that it is having authentication problems as demonstrated below.
+
+```console
+root@rook-ceph-mon0-rc568:/# ceph mon_status --cluster=rook --conf=/var/lib/rook/rook/rook.config --keyring=/var/lib/rook/rook/client.admin.keyring
+2017-10-11 19:47:24.894698 7f9b85919700  0 monclient(hunting): authenticate timed out after 300
+2017-10-11 19:47:24.894698 7f9b85919700  0 monclient(hunting): authenticate timed out after 300
+2017-10-11 19:47:24.894757 7f9b85919700  0 librados: client.admin authentication error (110) Connection timed out
+2017-10-11 19:47:24.894757 7f9b85919700  0 librados: client.admin authentication error (110) Connection timed out
+[errno 110] error connecting to the cluster
+```
+
+### Solution
+This is a common problem when reinitializing the Rook.io cluster and the local directory used for persistence has not been purged. This directory is the same directory reference by the `dataDiskHostPath` setting in the cluster CRD and is typically set to /var/lib/rook. To remedy simply shutdown all components of Rook.io and then delete the contents of /var/lib/rook or the directory specified by `dataDiskHostPath` on each of the hosts in the cluster. This time when the cluster CRD is applied, the rook-operator should start all the Pods as expected.
+
+## Troubleshooting Techniques
```

Comment:
what if we move this section to the top so people know where to start? I don't see it in the links at the top of the page either